tests/services/pids/providers/test_datacite_pid_provider.py

```diff
@@ -47,7 +47,7 @@ def record_w_links(running_app, minimal_record):
     draft = service.create(system_identity, minimal_record)
     record = service.publish(system_identity, draft.id)
 
-    return record
+    return record.to_dict()
```

Comment:
We were passing wrong types to the service provider. This was discovered due to the `_original` object of the post_dump processors